.github/workflows/test.yml

```diff
@@ -0,0 +1,121 @@
+name: Run instrumentation tests
+
+on:
+  push:
+    paths:
+      - '*.gradle'
+      - '*.gradle.kts'
+      - '**/*.gradle'
+      - '**/*.gradle.kts'
+      - '*.properties'
+      - '**/*.properties'
+      - '**/*.kt'
+      - '**/*.xml'
+      - '.github/workflows/test.yml'
+
+jobs:
+  instrumentation_tests:
+    runs-on: macos-11
+    strategy:
+      fail-fast: false
+      matrix:
+        android_target: [33, 34]
```

Comment:
I like the idea of testing for 13 in the 14 branch, to make sure any changes we make here don't break stuff / are backwards compatible. As much as possible, at least. We'll try.